contracts/FiatTokenV2_2/contracts/util/SignatureCheckerFiatProxy.sol

```diff
@@ -25,9 +25,9 @@ import { IERC1271 } from "../interface/IERC1271.sol";
  * @dev Signature verification helper that can be used instead of `ECRecover.recover` to seamlessly support both ECDSA
  * signatures from externally owned accounts (EOAs) as well as ERC1271 signatures from smart contract wallets.
  *
- * Adapted from https://github.com/OpenZeppelin/openzeppelin-contracts/blob/21bb89ef5bfc789b9333eb05e3ba2b7b284ac77c/contracts/utils/cryptography/SignatureChecker.sol
+ * Adapted from https://github.com/OpenZeppelin/openzeppelin-contracts/blob/21bb89ef5bfc789b9333eb05e3ba2b7b284ac77c/contracts/utils/cryptography/SignatureCheckerFiatProxy.sol
```

Comment:
There was a conflict with another SignatureChecker from EAS libs, so renamed this to avoid issues (This one compiles with 0.6.x whereas EAS compiles with 0.8.27)